SECURITY.md

```diff
@@ -0,0 +1,15 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+To report security issues send an email to security@acinq.fr (not for support).
+
+The following keys may be used to communicate sensitive information to developers:
+
+| Name                | Fingerprint                                       |
+|---------------------|---------------------------------------------------|
+| Pierre-Marie Padiou | 6AA4 5A4C 209A 2D30 64CF 66BE E434 ED29 2E85 643A |
+| Fabrice Drouin      | C25A 288A 842E AF7A A5B5 303F 7A73 FE77 DE2C 4027 |
+
+You can import a key by running the following command with that individual’s fingerprint: `gpg --recv-keys "<fingerprint>"`.
```

Comment:
In my case `gpg --recv-keys <fingerprint>` is not enough to import the keys because it fails to fetch the key for the fingerprint, adding the key server option fixed it:
`gpg --keyserver https://acinq.co/pgp/drouinf.asc --recv-keys C25A288A842EAF7AA5B5303F7A73FE77DE2C4027`

I couldn't find the key from @pm47 on `acinq.co/pgp` and using `https://api.github.com/users/pm47/gpg_keys` as key server doesn't work.

I propose to upload the keys and provide instructions to import the keys correctly.

Reply:
Good catch, I'll dive into that (sounds like a rabbit hole)

Reply:
> I couldn't find the key from @pm47 on `acinq.co/pgp` and using `https://api.github.com/users/pm47/gpg_keys` as key server doesn't work.

It's available on: https://acinq.co/pgp/padioupm.asc